<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConnectionsForm</class>
 <widget class="QWidget" name="ConnectionsForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>392</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="labelJiEpic">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Jira Epic:</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelIssToJira">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>170</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>new issus to Jira</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditBbRepo">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>130</y>
     <width>201</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxIssToJi">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>170</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>yes</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>no</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="titleSettings">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>0</y>
     <width>321</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-weight: bold;</string>
   </property>
   <property name="text">
    <string>Tracker Connection</string>
   </property>
   <property name="textFormat">
    <enum>Qt::PlainText</enum>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxCommToBitb">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>270</y>
     <width>51</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>yes</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>no</string>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="lineEditJiEpic">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>90</y>
     <width>201</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelDestrOnBitb">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>330</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>destructive on Bitbucket</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditJiProj">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>50</y>
     <width>201</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelCommToJira">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>240</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>comments to Jira</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelIssLogIssue">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>360</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Log issue ID (Jira)</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelCommToBitb">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>270</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>comments to BitBucket</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxDestrOnBitb">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>330</y>
     <width>51</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>yes</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>no</string>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="lineEditLogIssue">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>360</y>
     <width>201</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelDestrOnJi">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>300</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>destructive on Jira</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelJiProj">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Jira Project:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxDestrOnJi">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>300</y>
     <width>51</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>yes</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>no</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="labelBbRepo">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Bitbucket repository</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelIssToBitb">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>210</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>new issus to Bitbucket</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxCommToJi">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>240</y>
     <width>51</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>yes</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>no</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="comboBoxIssToBitb">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>210</y>
     <width>51</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>yes</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>no</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="pushButtonSave">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>410</y>
     <width>91</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>save changes</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonSaveAsNewConn">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>410</y>
     <width>151</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>save as new connection</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonDeleteConn">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>410</y>
     <width>51</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>delete </string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonReset">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>410</y>
     <width>41</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>reset</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonAddFieldMapping">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>440</y>
     <width>151</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>add new field mapping</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
